### ISSUE TITLE
Cannot ignore file in Knip analysis

### ISSUE THREAD

**Issue by tal-rofe** [webpro/knip#205]
I have a file which makes Knip to fail. I want to ignore that file. So I did:
```json
{
	"$schema": "./node_modules/knip/schema.json",
	"ignore": ["./bin/sudo-cli.js"]
}
```
The file is indeed `./bin/sudo-cli.js`. And error is:
```
Unresolved imports (1)
../dist/index.cjs  bin/sudo-cli.js
```

So I cannot ignore that error.

**Reproduce**
Clone this project: https://github.com/tal-rofe/sudo-cli/tree/use-spawn
Checkout to `use-spawn` branch. Run `pnpm i`, then run `pnpm knip`. You also need to add `"ignore": ["./bin/sudo-cli.js"]` to `knip.json` file to reproduce.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.19.5. See [Release 2.19.5](https://github.com/webpro/knip/releases/tag/2.19.5) for release notes._

**Comment by webpro** [comment]
Now unresolved imports that are in `.gitignore` should be ignored by default.

By the way, you can now remove `cz-vinyl` from the list of ignored dependencies. And `commitlint` is an alias of `@commitlint/cli` so you could use them consistently and remove another from that list :)
